spec/integration/knife/config_list_profiles_spec.rb

```diff
@@ -82,9 +82,9 @@
         chef_server_url = "https://example.com/organizations/testorg"
       EOH
       it { is_expected.to eq <<~EOH.delete("#") }
-         Profile  Client    Key                  Server                                   #
-        ----------------------------------------------------------------------------------#
-        *default  testuser  ~/.chef/testkey.pem  https://example.com/organizations/testorg#
+         Profile  Client    Key                  Server                                    #
+        \e[32m--------------------------------------------------------------------------------\e[0m#
```

Comment:
- 2 `"-"` chars got remove due to table border padding applied with one space instead of two.
- Earlier color code added it is not working table handle border-color out of the box.